### ISSUE TITLE
SkyDrop Dead after FW flash

### ISSUE THREAD

**Issue by vittuhy** [fhorinek/SkyDrop#471]
Ahoj,
my new Skydrop seems to be dead after flashing. I contacted your support a few days ago regarding Bluetooth issues:

- contact form message #917826
- BT Pan 1026 v4.0
- Build 5276

BT icon was not flashing on the main screen and was not visible to any app like Flyskyhy or XCTrack (I tried many devices with many iOS and Android versions). Your support team was unresponsive for a few days so I decided to downgrade to 5205 which works fine on a colleague's device. I followed the flashing manual, new FW was found by the device, unpacked and updated, then the device switched off and never rebooted again. Now it is completely dead and cannot be enabled, when connected to the laptop no mass storage appears. Is there any way how to proceed with some hard reset or how to wake the SkyDrop up?

@fhorinek, I need to solve this up as soon as possible since I do not have any backup device, could you please get in touch with me to discuss further steps? My phone number is filled in my SkyBean account at your website under vituhy@gmail.com.

Diky, Vitek

**Comment by fhorinek** [comment]
Newest SkyDrops might use different MCU then the old ones.
Older firmware do not support new MCU.

You need to update it to newest version
- disassemble the device
- unplug the battery
- remove sd card
- place UPDATE.FW and UPDATE.EE from https://vps.skybean.eu/configurator/fw/00005276/ to the sd card
- eject the sdcard
- put it back to skydrop
- re-plug the battery

**Comment by vittuhy** [comment]
Dobry den @fhorinek 
following your steps SkyDrop is now running the latest FW and is booting normally. Nevertheless, after enabling Bluetooth no BT flashing icon appears on the home screen and the device is not searchable by any phone. Any ideas? Thanks, Vit

Video> https://www.veed.io/view/6c7f0b48-f316-40d9-9a24-937f1a0eacfa/showcase?sharingWidget=true

**Comment by vittuhy** [comment]
After a factory reset the Bluetooth was running well for a while and I managed to connect the device to the smart phone. Then I left the device shut down for hour or so, and after enabling it the Bluetooth was not available again. I tried to do the factory reset again but this time it did not help. Seems like somethings wrong with the Bluetooth module. Is it possible to send you the device for further diagnostic?
